File: 02-programacion-basica/13-biblioteca-estandar/java/ejemplos-guiados/BibliotecaEstandar.java
import java.time.LocalDate;
import java.util.Arrays;
import java.util.Random;

public class BibliotecaEstandar {
    public static void main(String[] args) {
        int[] valores = {8, 3, 12, 5};

        // La biblioteca estándar evita reimplementar operaciones comunes.
        Arrays.sort(valores);
        Random random = new Random(42); // semilla fija para obtener resultados reproducibles

        System.out.println("Raíz de 81: " + Math.sqrt(81));
        System.out.println("Fecha: " + LocalDate.now());
        System.out.println("Aleatorio reproducible: " + random.nextInt(10));
        System.out.println("Ordenados: " + Arrays.toString(valores));
    }
}

File: 02-programacion-basica/13-biblioteca-estandar/scala/ejemplos-guiados/BibliotecaEstandar.scala
import java.time.LocalDate
import scala.util.Random

object BibliotecaEstandar:
  def main(args: Array[String]): Unit =
    val valores = Array(8, 3, 12, 5)
    val ordenados = valores.sorted
    val random = new Random(42)

    println(s"Raíz de 81: ${math.sqrt(81)}")
    println(s"Fecha: ${LocalDate.now()}")
    println(s"Aleatorio reproducible: ${random.nextInt(10)}")
    println(s"Ordenados: ${ordenados.mkString("[", ", ", "]")}")
